script/Ciel/Day 9/09_01_CIEL09_2.txt

```diff
@@ -524,42 +589,44 @@ She's reprimanding me because I put myself in danger.
 [sha:3caed3dccf66f413378ac9be983e997777020400]{
 -- Page 140, Offset 22489.
 -- 「シエル先輩、あの―――」
-"Ciel-senpai, I―――"
+"Ciel-senpai, um―――"
 }
 [offset:22490]{
 -- Page 140, Offset 22490.
 -- ……だめだ。
-I can't say it.
+It's no use.
```

Comment:
I'd almost just prefer to leave this at a solitary "Damnit...", or something.
Very much don't like how 'It's no use' ties these sentences together.